src/model.jl

```diff
@@ -8,90 +9,180 @@ mutable struct Model
+    geography::Array{Int8,2}            # Monthly land-sea-snow/sea ice distribution: depends on initial geography input if compute_sea_ice_extent is true
+    solar::Array{Float64,2}             # Solar irradiance: depends on the orbital parameters
+    compute_sea_ice_extent::Bool        # Attribute for deciding whether sea ice extent should be calculated for the geography or read from given maps
+    year::Int64    
   end
 
-  function Model(mesh, num_steps_year; co2_concentration = 315.0, compute_albedo = false) # co2_concentration in [ppm], default value from year 1950
+  function Model(mesh, num_steps_year; co2_concentration = 315.0, compute_albedo = false, compute_sea_ice_extent = false, sea_ice_extent_year = 1979) # co2_concentration in [ppm], default value from year 1950
     @unpack nx,ny = mesh
     
     # Constants
     radiative_cooling_feedback = 2.15   #[W/m^2/°C]: sensitivity of the seasonal cycle and annual change in the forcing agents
-
+  
     radiative_cooling_co2 = calc_radiative_cooling_co2(co2_concentration)
     
     # Read parameters
     
     geography = read_geography(joinpath(@__DIR__, "..", "input", "The_World.dat"),nx,ny)
+    
+    compute_sea_ice_extent = compute_sea_ice_extent
+    year = sea_ice_extent_year
 
-  
     compute_albedo = compute_albedo
     if compute_albedo == false
       albedo = read_albedo(joinpath(@__DIR__, "..", "input", "albedo.dat"),nx,ny)
     else
-
+  
       albedo = calc_albedo(geography,nx,ny)
     end 
 
 
+  
     diffusion_coeff = calc_diffusion_coefficients(geography,nx,ny)
     heat_capacity, tau_land, tau_snow, tau_sea_ice, tau_mixed_layer = calc_heat_capacity(geography,radiative_cooling_feedback) # TODO: Remove unused variables
-
+  
     co_albedo = 1.0 .- albedo
     
     ecc = eccentricity(1950)
     ob = obliquity(1950)
     per = perihelion(1950)
-    solar_forcing = calc_solar_forcing(co_albedo, ecc=ecc, ob=ob, per=per) #TODO: Add arguments [nx, ny, num_steps_year]
+     
+    
+    lambda,solar = calc_solar_forcing_params(nlatitude=ny, ntimesteps=num_steps_year, ecc=ecc, ob=ob, per=per)
+    solar_forcing = calc_solar_forcing(co_albedo, solar,nx,ny,num_steps_year)
   
     
-    return Model(diffusion_coeff, heat_capacity, albedo, solar_forcing, radiative_cooling_co2, radiative_cooling_feedback,co_albedo,compute_albedo)
-end
-
-function set_co2_concentration!(model, co2_concentration)
+    return Model(diffusion_coeff, heat_capacity, albedo, solar_forcing, radiative_cooling_co2, radiative_cooling_feedback,co_albedo,compute_albedo,geography,solar, compute_sea_ice_extent,year)
+  end
+  
+  function set_co2_concentration!(model, co2_concentration)
     model.radiative_cooling_co2 = calc_radiative_cooling_co2(co2_concentration)
-end
-
-
+  end
+  
+  
+  # Gives the possibility to interactively decide which parameters should be updated in a simulation
```

Comment:
```suggestion
"""
  update_model()
  Gives the possibility to interactively decide which parameters should be updated in a simulation
"""
```
Please add this as julia docstring